components/TARGET_PSA/TARGET_TFM/COMPONENT_NSPE/interface/src/tfm_ns_lock_rtx.c

```diff
@@ -87,3 +87,17 @@ bool tfm_ns_lock_get_init_state()
     return ns_lock.init;
 }
 
+bool tfm_ns_lock_get_lock_state()
+{
+    if (ns_lock.init) {
+        /* osRtxMutexId() is defined in rtx_lib.h. Inclusion of rtx_lib.h needs to
+         * include extra header file like RTE_Components.h. To avoid inclusion error
+         * or inclusion of more irrelevant header files, we expand osRtxMutexId()
+         * in-place to get mutex lock count straight. */
+        //os_mutex_t *mutex = osRtxMutexId(ns_lock.id);
```

Comment:
remove dead code

Reply:
There's an on-going change [Interface: Refactor the NS interface](https://review.trustedfirmware.org/c/trusted-firmware-m/+/1460) in TFM repo. The change to TF-M NS interface and related code in this PR are temporary. Just ignore it and wait for TF-M NS interface to be confirmed.